spec.emu

```diff
@@ -196,9 +196,9 @@ contributors: Ujjwal Sharma, Younies Mahmoud
       <emu-alg>
         1. Let _sign_ be ! DurationRecordSign(_record_).
         1. For each row of <emu-xref href="#table-duration-components"></emu-xref>, except the header row, in table order, do
-          1. Let _valueSlot_ be the Value Slot value.
-          1. Let _v_ be value of the _valueSlot_ slot of _record_.
-          1. If 𝔽(_v_) is not finite, return *false*.
+          1. Let _valueSlot_ be the Value Slot value of the current row.
+          1. Let _v_ be _record_.[[&lt;_valueSlot_&gt;]].
+          1. Assert: 𝔽(_v_) is finite.
```

Comment:
Is this a normative change? Although I'm not sure if we should cater to infinite values...

Reply:
Duration records in `Intl.DurationFormat` are always created through `ToDurationRecord`, which rejects infinite values (through the `IsIntegralNumber` check). I only performed this change, because we can't currently have infinite values, so it seems better to me to use an assertion.